inlayHints.py

```diff
@@ -0,0 +1,111 @@
+from .protocol import InlayHint, InlayHintRequestParams, InlayHintResponse
+from html import escape as html_escape
+from LSP.plugin import Request
+from LSP.plugin import Session
+from LSP.plugin.core.protocol import Point
+from LSP.plugin.core.registry import windows
+from LSP.plugin.core.types import debounced
+from LSP.plugin.core.types import FEATURES_TIMEOUT
+from LSP.plugin.core.typing import List, Optional, Tuple
+from LSP.plugin.core.views import point_to_offset
+from LSP.plugin.core.views import text_document_identifier
+import sublime
+import sublime_plugin
+
+
+def inlay_hint_to_phantom(view: sublime.View, hint: InlayHint) -> sublime.Phantom:
+    html = """
+    <body id="lsp-typescript-inlay-hints">
+        <style>
+            body {{
+                padding: 0;
+                margin: 0px;
+                border: 0px;
+            }}
+
+            #lsp-typescript-inlay-hints .inlay-hint {{
+                color: color(var(--foreground) alpha(0.6));
+                background-color: color(var(--foreground) alpha(0.08));
+                border-radius: 4px;
+                padding-left: 4px;
+                padding-right: 4px;
+                font-size: 0.9em;
+            }}
+        </style>
+        <div class="inlay-hint">
+            {label}
+        </div>
+    </body>
+    """
+    point = Point.from_lsp(hint['position'])
+    region = sublime.Region(point_to_offset(point, view))
+    label = html_escape(hint["text"])
+    html = html.format(label=label)
+    return sublime.Phantom(region, html, sublime.LAYOUT_INLINE)
+
+
+def session_by_name(view: sublime.View, session_name: str) -> Optional[Session]:
```

Comment:
ideally a function like `session_by_name` should be exposed from LSP, 
Currently this function can easily break if something changes on the LSP side. (example, move the `windows` variable to a different file).

@rwols @rchl what do you guys think?

Reply:
Whatever works for now. We control both sides so hopefully nothing will break without us catching it first.